docs/source/algorithms/CreateDetectorTable-v1.rst

```diff
@@ -0,0 +1,36 @@
+.. algorithm::
+
+.. summary::
+
+.. relatedalgorithms::
+
+.. properties::
+
+Description
+-----------
+
+Creates a table workspace of detector information for a given matrix or peaks workspace.
+
+Usage
+-----
+
+**Example**
+
+.. testcode:: CreateDetectorTableExample
+
+   ws = CreateSampleWorkspace()
+
+   ws_detectors = CreateDetectorTable(ws)
+
+   # Print the first row of the table
+   print(ws_detectors.row(0))
+
+Output:
+
+.. testoutput:: CreateDetectorTableExample
+
+  {'Index': 0.0, 'Phi': 0.0, 'Monitor': 'no', 'Spectrum No': 1, 'R': 5.0, 'Detector ID(s)': '100', 'Theta': 0.0}
```

Comment:
```suggestion
  {'Index': 0.0, 'Spectrum No': 1, 'Detector ID(s)': '100', 'R': 5.0, 'Theta': 0.0, 'Phi': 0.0, 'Monitor': 'no'}
```
This should fix the failing doctest

Reply:
Oh I literally just pushed a change for this. I thought the failure was due to dictionaries being unordered so the output order varied. I've changed the way the output is created so the order is fixed.

Reply:
Your solution is better. this was a quick fix but much more fragile than yours.